### ISSUE TITLE
Define a variable

### ISSUE THREAD

**Issue by mayisa87** [DorinC263/Hangman#3]
https://github.com/DorinC263/Hangman/blob/f5027f1836326b824675e37711c57ab3900ecf32/Program.cs#L58

If you define a variable Color foreground you could set the value to the color you want and call the set method only once after your if Statement.

**Comment by mayisa87** [comment]
#1 merged into one issue :)
